src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -170,6 +171,13 @@ export class CalciteRadioButton {
    */
   @Event() calciteRadioButtonChange: EventEmitter;
 
+  /** Fires when the checked property changes.  This is an internal event used for styling purposes only.
+   * Use calciteRadioButtonChange or calciteRadioButtonGroupChange for responding to changes in the checked value for forms.
+   * @internal
+   */
+  @Event() calciteRadioButtonCheckedChange: EventEmitter;
+
+  /** Fires when the radio button is either focused or blurred. */
```

Comment:
Would love feedback on the docs wording here.  It should be emphasized to developers using this component that this new `checkedChange` event should not be used to update form state because of the risk of infinite loops.  The infinite loop is likely to occur when setting the `checked` property of a radio-button based on listening for this event, because this event emits in the `checked` watcher.